plugins/node/opentelemetry-winston-transport/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+## [0.1.0]() ()
+
+### Features
+
+* Add Winston OpenTelemetry Transport ([#](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/))
```

Comment:
The changelog can be empty on a new package, it'll be filled automatically on the first release.